contributing/code-style-and-quality-pull-requests.md

```diff
@@ -0,0 +1,93 @@
+---
+title:  "Apache Flink Code Style and Quality Guide — Pull Requests & Changes"
+---
+
+{% include code-style-navbar.md %}
+
+{% toc %}
+
+**Rationale:** We ask contributors to put in a little bit of extra effort to bring pull requests into a state that they can be more easily and more thoroughly reviewed. This helps the community in many ways:
+
+
+* Reviews are much faster and thus contributions get merged sooner.
+* We can ensure higher code quality by overlooking fewer issues in the contributions.
+* Committers can review more contributions in the same time, which helps to keep up with the high rate of contributions that Flink is experiencing
+
+Please understand that contributions that do not follow this guide will take longer to review and thus will typically be picked up with lower priority by the community. That is not ill intend, it is due to the added complexity of reviewing unstructured Pull Requests.
+
+
+## 1. JIRA issue and Naming
+
+Make sure that the pull request corresponds to a [JIRA issue]([https://issues.apache.org/jira/projects/FLINK/issues](https://issues.apache.org/jira/projects/FLINK/issues)).
+
+Exceptions are ****hotfixes****, like fixing typos in JavaDocs or documentation files.
+  
+
+Name the pull request in the form `[FLINK-XXXX] [component] Title of the pull request`, where `FLINK-XXXX` should be replaced by the actual issue number. The components should be the same as used in the JIRA issue.
```

Comment:
the majority of PRs/commits that I see do not have a space between `][`

Reply:
+1

Reply:
Any version works, as long as we are consistent.
I am fine with "no space in between".

Reply:
I also see the "no space in between" variant more often. I'll change it